src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,28 @@
+import { NavLink } from 'react-router-dom';
+
+import styles from './Navigation.module.css';
+
+import { HOME_LABEL, ABOUT_LABEL } from '../constants';
+
+const Navigation = () => {
+  const checkIsActive = ({ isActive }) => (isActive ? { textDecoration: 'underline' } : undefined);
```

Comment:
awesome! :)
that being said make sure you always return the same type of data so `isActive ? { textDecoration: 'underline' } : {}`

Reply:
Thank you. I have changed it.